### PR TITLE
Enable slack notifications per entity

### PR DESCRIPTION
**Pull request by renzodgc**
Move Slack Configuration from a global config to each area/camera

### CODE REVIEW

File: api/models/config_keys.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class NotificationConfig(SnakeModel):
     violationThreshold: Optional[int] = Field(0, example=100)
     notifyEveryMinutes: Optional[int] = Field(0, example=15)
     emails: Optional[str] = Field("", example='john@email.com,doe@email.com')
+    enableSlackNotifications: Optional[str] = Field(False, example=False)
```
Comment (pgrill) [inline api/models/config_keys.py:27]: Maybe we can declare it as a bool field.

Comment (renzodgc) [reply api/models/config_keys.py:27]: I think it needs to be a string to be written in the config file

Comment (pgrill) [reply api/models/config_keys.py:27]: :thinking: But the daily report is declared as `bool` and it works.
If we declare it as a string, the field definition should be `Field("False", example="False")`

**Review by mats-claassen — APPROVED** [review]

File: api/models/config_keys.py (hunk 1 above)
Comment (renzodgc) [reply api/models/config_keys.py:27]: You were right :), I got mixed up

**Review by pgrill — APPROVED** [review]

**Pull request merged**